framework/generated/generated_vulkan_struct_to_json.cpp

```diff
@@ -431,15 +438,11 @@ void FieldToJson(nlohmann::ordered_json& jdata, const Decoded_StdVideoEncodeH264
         FieldToJson(jdata["flags"], meta_struct.flags, options);
         FieldToJson(jdata["first_mb_in_slice"], decoded_value.first_mb_in_slice, options);
         FieldToJson(jdata["slice_type"], decoded_value.slice_type, options);
-        FieldToJson(jdata["idr_pic_id"], decoded_value.idr_pic_id, options);
-        FieldToJson(jdata["num_ref_idx_l0_active_minus1"], decoded_value.num_ref_idx_l0_active_minus1, options);
-        FieldToJson(jdata["num_ref_idx_l1_active_minus1"], decoded_value.num_ref_idx_l1_active_minus1, options);
-        FieldToJson(jdata["cabac_init_idc"], decoded_value.cabac_init_idc, options);
-        FieldToJson(jdata["disable_deblocking_filter_idc"], decoded_value.disable_deblocking_filter_idc, options);
         FieldToJson(jdata["slice_alpha_c0_offset_div2"], decoded_value.slice_alpha_c0_offset_div2, options);
         FieldToJson(jdata["slice_beta_offset_div2"], decoded_value.slice_beta_offset_div2, options);
         FieldToJson(jdata["reserved1"], decoded_value.reserved1, options);
-        FieldToJson(jdata["reserved2"], decoded_value.reserved2, options);
+        FieldToJson(jdata["cabac_init_idc"], decoded_value.cabac_init_idc, options);
+        FieldToJson(jdata["disable_deblocking_filter_idc"], decoded_value.disable_deblocking_filter_idc, options);
         FieldToJson(jdata["pWeightTable"], meta_struct.pWeightTable, options);
```

Comment:
Fields are going away after they have been published?

Reply:
Ditto the above comment.